test/integrations/destinations/facebook_pixel/dataDelivery/business.ts

```diff
@@ -0,0 +1,357 @@
+import { generateMetadata, generateProxyV1Payload } from '../../../testUtils';
+import { ProxyV1TestData } from '../../../testTypes';
+import { VERSION } from '../../../../../src/v0/destinations/facebook_pixel/config';
+
+const testData = [
+  '{"user_data":{"external_id":"c58f05b5e3cc4796f3181cf07349d306547c00b20841a175b179c6860e6a34ab","client_ip_address":"32.122.223.26","client_user_agent":"Mozilla/5.0 (iPhone; CPU iPhone OS 15_5 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.5 Mobile/15E148 Safari/604.1"},"event_name":"Checkout Step Viewed","event_time":1654772112,"event_source_url":"https://www.my.kaiser.com/checkout","event_id":"4f002656-a7b2-4c17-b9bd-8caa5a29190a","custom_data":{"checkout_id":"26SF29B","site":"www.my.kaiser.com","step":1}}',
```

Comment:
what is the difference between testData and testData2

Reply:
event_time is different

Reply:
it is very hard to figure out, can you use a common JSON and then replace the field of interest, and then convert it to string using JSON Stringify

Reply:
Actually having that `event_time` different is not serving any purpose. So updated the tests with the same event_time values